server/Entities/Models/Category.cs

```diff
@@ -0,0 +1,15 @@
+using Entities.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Entities.Models
+{
+    public class Category
+    {
+        public int CategoryId { get; set; }
+        public string CategoryName { get; set; }
```

Comment:
1. it is entity Category, it is obvious that properties belongs Category, so just Id and Name
2. Instead, it is unclear, about category - Category of what? it seems thant name of entity does't tell the goal of using

Reply:
changed naming : category => NewsCategory